app/exec/extension/_lib/vsix-writer.ts

```diff
@@ -184,18 +185,28 @@ export class VsixWriter {
 						if (!builder.files[path].content) {
 							let readFilePromise = promisify(readFile)(path).then(result => {
 								vsix.file(itemName, result);
+								seenPartNames.add(itemName);
 								if ((builder.files[path] as any)._additionalPackagePaths) {
 									for (const p of (builder.files[path] as any)._additionalPackagePaths) {
-										vsix.file(p, result);
+										let additionalItemName = toZipItemName(p);
```

Comment:
Why are we using toZipItemName now when we weren't before?

Reply:
I think that was a bug/oversight. When I was debugging this issue, I saw some errors reported by zip when unzipping the file and this appeared to fix that.

Reply:
and by bug/oversight, I meant it was left off before and I fixed it now